h2o-algos/src/main/java/hex/pca/PCA.java

```diff
@@ -58,26 +56,28 @@
  //   int numCPUs= H2O.NUMCPUS;   // proper way to get number of CPUs.
     double p = hex.util.LinearAlgebraUtils.numColsExp(_train,true);
     double r = _train.numRows();
-    long mem_usage =
-            _parms._pca_method == PCAParameters.Method.GramSVD ? (long)(hb._cpus_allowed * p*p * 8/*doubles*/ *
+    boolean useGramSVD = _parms._pca_method == PCAParameters.Method.GramSVD;
+    boolean usePower = _parms._pca_method==PCAParameters.Method.Power;
+
+    long mem_usage = (useGramSVD || usePower) ? (long)(hb._cpus_allowed * p*p * 8/*doubles*/ *
                     Math.log((double)_train.lastVec().nChunks())/Math.log(2.)) : 1; //one gram per core
-    long mem_usage_w = _parms._pca_method == PCAParameters.Method.GramSVD ? (long)(hb._cpus_allowed * r*r *
+    long mem_usage_w = (useGramSVD || usePower) ? (long)(hb._cpus_allowed * r*r *
```

Comment:
I am little bit lost here. Why is Power method taking same amount of memory? I thought the whole point of power method was you don't have to form the gram?